tests/test_tqdm_utils.py

```diff
@@ -0,0 +1,17 @@
+from unittest.mock import patch
+
+from transformers import AutoModel
+from transformers.utils.tqdm_utils import set_progress_bar_enabled
+
+
```

Comment:
```suggestion
from transformers import AutoModel
from transformers.utils.tqdm_utils import set_progress_bar_enabled
from transformers.testing_utils import  require_torch

@require_torch
```

pytorch tests need to be designated as such, since otherwise tf/flax CI will fail as it doesn't have pytorch.

you will probably will need to run `make fixup` and push to keep the autoformatter happy.

Reply:
I think my 2 suggestions overlap, I hope it's easy to sort out.